.travis/custom_release.sh

```diff
@@ -6,15 +6,16 @@ if [ "${TRAVIS_BRANCH}" = "master" ]; then
     echo
     echo
     echo "PUSHING ci-beta"
-    .travis/release.sh "ci-beta"
-    rm -rf dist/.git
+    BUILD_BETA=true .travis/release.sh "ci-beta"
```

Comment:
I need to check and see if this does an npm build or if thats done prior to this which would make this inefficient.